src/pages/VenueLandingPage/VenueLandingPage.tsx

```diff
@@ -123,7 +124,9 @@ export const VenueLandingPage: React.FunctionComponent<VenueLandingPageProps> =
         : venueEntranceUrl(venueId);
   };
 
-  const hasSecretForm = isTruthy(venue.showSecretPasswordForm);
+  const hasPassword = isTruthy(
+    venue.access?.includes(VenueAccessType.Password)
+  );
```

Comment:
`.includes` already returns a boolean, and we can default if `undefined`. Better to be explicit:

```suggestion
  const hasPassword = venue.access?.includes(VenueAccessType.Password) ?? false;
```

Reply:
It might also be worth calling this something like `requiresVenuePassword` to be explicit/obvious in what it's intent is.